modules/metering-troubleshooting.adoc

```diff
@@ -10,10 +10,173 @@ A common issue with metering is pods failing to start. Pods might fail to start
+----
+
+. Check the `ReportDataSource` resources where the `NEWEST METRIC` is less than the report end date:
++
+[source,terminal]
+----
+$ oc get reportdatasource
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                         EARLIEST METRIC        NEWEST METRIC          IMPORT START           IMPORT END             LAST IMPORT TIME       AGE
+...
+node-allocatable-cpu-cores                   2020-04-23T09:14:00Z   2020-08-31T10:07:00Z   2020-04-23T09:14:00Z   2020-10-15T17:13:00Z   2020-12-09T12:45:10Z   230d
+node-allocatable-memory-bytes                2020-04-23T09:14:00Z   2020-08-30T05:19:00Z   2020-04-23T09:14:00Z   2020-10-14T08:01:00Z   2020-12-09T12:45:12Z   230d
+...
+pod-usage-memory-bytes                       2020-04-23T09:14:00Z   2020-08-24T20:25:00Z   2020-04-23T09:14:00Z   2020-10-09T23:31:00Z   2020-12-09T12:45:12Z   230d
+----
+
+. Check the health of the `reporting-operator` pod for a high number of pod restarts:
++
+[source,terminal]
+----
+$ oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  READY   STATUS    RESTARTS   AGE
+hive-metastore-0                      2/2     Running   0          45h
+hive-server-0                         3/3     Running   0          45h
+metering-operator-7b9d7958d7-z6n46    2/2     Running   0          46h
+presto-coordinator-0                  2/2     Running   1          45h
+reporting-operator-84f7c9b7b6-fr697   2/2     Running   542        8d <1>
+----
+<1> The `reporting-operator` pod is restarting at a high rate.
+
+. Check the `reporting-operator` pod for an `OOMKilled` termination:
++
+[source,terminal]
+----
+$ oc describe pod/reporting-operator-84f7c9b7b6-fr697
+----
++
+.Example output
+[source,terminal]
+----
+Name:         reporting-operator-84f7c9b7b6-fr697
+Namespace:    operator-metering
+Priority:     0
+Node:         ip-10-xx-xx-xx.ap-southeast-1.compute.internal/10.xx.xx.xx
+...
+   Ports:          8080/TCP, 6060/TCP, 8082/TCP
+   Host Ports:     0/TCP, 0/TCP, 0/TCP
+   State:          Running
+     Started:      Thu, 03 Dec 2020 20:59:45 +1000
+   Last State:     Terminated
+     Reason:       OOMKilled <1>
+     Exit Code:    137
+     Started:      Thu, 03 Dec 2020 20:38:05 +1000
+     Finished:     Thu, 03 Dec 2020 20:59:43 +1000
+----
+<1> The `reporting-operator` pod was terminated due to OOM kill.
+
+[discrete]
+[id="metering-check-and-increase-memory-limits_{context}"]
+=== Increasing the reporting-operator pod memory limit
+
+If you are experiencing an increase in pod restarts and OOM kill events, you can check the current memory limit set for the `reporting-operator` pod. Increasing the memory limit allows the `reporting-operator` pod to update the report data sources. If necessary, increase the memory limit in your `MeteringConfig` resource by 25-50%.
```

Comment:
Same feedback here after looking over your PR from this morning. Do we want to substitute `reporting-operator` -> Reporting Operator?

Reply:
@timflannagan @adellape I think for general usage/mention, "Reporting Operator pod" works. 👍  

When saying to check specific information regarding the pod, WDYT of the following example:

> Check the current memory limits of the `reporting-operator` `Pod` resource:

Reply:
That sounds fine to me, but I thought we weren't capitalizing Pod in any circumstances anymore?

Reply:
Pod can be capitalized if we're referring to a specific API object. So it's in backticks followed by the type of object. In this case, "`Pod` resource".